pkg/config/parameters.go

```diff
@@ -689,6 +737,47 @@ func (p *MetricsParameters) Validate() error {
 	return nil
 }
 
+func (t *Tracing) Validate() error {
+	if t == nil {
+		return nil
+	}
+	if t.OverallSampling == 0 && t.MaxPathTagLength == 0 && t.ExtensionService == "" && t.CustomTags == nil {
+		return nil
+	}
```

Comment:
Same comment here, I think if Tracing is non-nil, then we should require ExtensionService, so just drop these lines and let the rest of the logic catch errors.

Reply:
I see that we're somewhat inconsistent in our existing code here, but I think it makes the most sense to use a `*Tracing` field that can be nil on the top-level Parameters struct, and not initialize it in the `Defaults` function.